From ecca557e702ec6d1e44ce18018ca65d8c3705407 Mon Sep 17 00:00:00 2001
Message-Id: <ecca557e702ec6d1e44ce18018ca65d8c3705407.1350297260.git.jdenemar@redhat.com>
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Thu, 11 Oct 2012 19:56:44 -0600
Subject: [PATCH] Remove some unused includes in QEMU code

https://bugzilla.redhat.com/show_bug.cgi?id=863115

The qemu monitor does not require qemu_conf.h, and the
qemu capabilities code actually wants bitmap.h

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
(cherry picked from commit c25746c2169918accd6b04c18a70e98145675faf)
---
 src/qemu/qemu_capabilities.c | 2 +-
 src/qemu/qemu_monitor.c      | 1 -
 src/qemu/qemu_monitor.h      | 1 -
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/qemu/qemu_capabilities.c b/src/qemu/qemu_capabilities.c
index f8b4631..ccc1dae 100644
--- a/src/qemu/qemu_capabilities.c
+++ b/src/qemu/qemu_capabilities.c
@@ -32,8 +32,8 @@
 #include "nodeinfo.h"
 #include "cpu/cpu.h"
 #include "domain_conf.h"
-#include "qemu_conf.h"
 #include "command.h"
+#include "bitmap.h"
 #include "virnodesuspend.h"
 
 #include <sys/stat.h>
diff --git a/src/qemu/qemu_monitor.c b/src/qemu/qemu_monitor.c
index 129734a..865c6c4 100644
--- a/src/qemu/qemu_monitor.c
+++ b/src/qemu/qemu_monitor.c
@@ -31,7 +31,6 @@
 #include "qemu_monitor.h"
 #include "qemu_monitor_text.h"
 #include "qemu_monitor_json.h"
-#include "qemu_conf.h"
 #include "virterror_internal.h"
 #include "memory.h"
 #include "logging.h"
diff --git a/src/qemu/qemu_monitor.h b/src/qemu/qemu_monitor.h
index 85f5e7a..26c04d7 100644
--- a/src/qemu/qemu_monitor.h
+++ b/src/qemu/qemu_monitor.h
@@ -29,7 +29,6 @@
 
 # include "qemu_capabilities.h"
 # include "domain_conf.h"
-# include "qemu_conf.h"
 # include "bitmap.h"
 # include "virhash.h"
 # include "json.h"
-- 
1.7.12.3

